gui/wxpython/gui_core/toolbars.py

```diff
@@ -304,6 +300,221 @@ def CreateSelectionButton(self, tooltip=_("Select graphics tool")):
         return button
 
 
+class AuiToolbarController(ToolbarController):
+    """Provides handling for aui.AuiToolBar widget"""
+
+    def _defineTool(self, name=None, icon=None, handler=None, item=wx.ITEM_NORMAL):
+        """Define tool"""
```

Comment:
Explain why it needs to be different

Reply:
The problem is that AUI does not have InsertTool.. As I get it right, it is quite a new method in wx which is probably the reason. Look at https://www.geeksforgeeks.org/wxpython-inserttool-function-in-wx-toolbar/.